modules/customizing-kourier-bootstrap-for-kourier-getaways.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: PROCEDURE
+[id="customizing-kourier-bootstrap-for-kourier-getaways_{context}"]
+= Customizing kourier-bootstrap for Kourier getaways
+
+The Envoy proxy component in Kourier handles inbound and outbound HTTP traffic for the Knative services. By default, Kourier contains Envoy bootstrap configuration in the `kourier-bootstrap` ConfigMap. You can change this configuration.
```

Comment:
Maybe add in parentheses "The Envoy proxy component (3scale-kourier-gateway)..."